cluster-scope/base/namespaces/thoth-test-core/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
Did you mean to add this namespace to the zero cluster [kustomization](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/zero/kustomization.yaml) ?

Reply:
thank you, I couldn't find the kustomize file before :see_no_evil: , I have updated it.